src/flamenco/runtime/program/fd_bpf_loader_serialization.h

```diff
@@ -3,9 +3,8 @@
 
 #include "../../fd_flamenco_base.h"
 #include "../../vm/fd_vm.h"
+#include "../fd_runtime.h"
```

Comment:
needed?

Reply:
good catch.  just need it in serialization.c and not the .h